packages/cli-lib/lang/en.lyaml

```diff
@@ -553,6 +553,7 @@ en:
         subcommands:
           create:
             describe: "Create a sandbox account"
+            enterAccountName: "Enter a unique name to reference your account: "
```

Comment:
I used the same line as the `hs init` step. It currently prompts the user for a name but sets the sandbox name as the default if the user hits enter. 

Taking Mark's [comment here](https://hubspot.slack.com/archives/C01GV708YH3/p1651516531083669?thread_ts=1651167455.710479&cid=C01GV708YH3) into consideration, should we skip that step and just set the sandbox name as the cli config name?